### ISSUE TITLE
[Enhancement] Export/Import map retain signature information

### ISSUE THREAD

**Issue by Bluedagger21** [exodus4d/pathfinder#801]
Currently, exporting and important a map does not retain signature information. Can this be enhanced to do so? 

Alternatively, and even additionally, could we implement a "copy" of the signature or intel list for a map to allow an effective "paste" into a different map?

**Comment by exodus4d** [comment]
Yeah, it´s a known "problem" with the current im/export feature. Unfortunately adding sigs to map export was not possible without much work to be done. Reason for this is, that an export just exports the map data already available in the browser (there is no server call necessary to get all the data for export). 
Having all signatures for all systems and all maps available in the browser would result in gigantic JSON responses (e.g. 5 maps * 80 systems * 10 signatures = 4000 signatures. And this for all clients.....).

But this has changes with the next release. :) So there is a high chance I will add signatures to export in the next few weeks.
